tests/integration/test_scaling_monolithic.py

```diff
@@ -12,9 +13,10 @@
 from pytest_operator.plugin import OpsTest
 
 METADATA = yaml.safe_load(Path("./charmcraft.yaml").read_text())
-APP_NAME = "tempo"
+APP_NAME = "tempo-coordinator"
```

Comment:
mimir's coordinator is also deployed as 'mimir' in the bundle. I'd not change this.